run-tests.sh

```diff
@@ -10,7 +10,7 @@
 export PYTEST_ADDOPTS='docs tests invenio_app'
 
 pydocstyle invenio_app tests docs && \
-isort invenio_app/*.py tests/*.py --check-only --diff && \
+isort invenio_app/*.py tests/*.py . --recursive --check-only --diff && \
```

Comment:
IIRC the `--recursive` flag was deprecated in version 5.0.0. See [here](https://github.com/timothycrosley/isort/blob/85e07d44cc49ebbe605c24eefc6779968db3d684/docs/upgrade_guides/5.0.0.md#L29).

I think introducing the flag might cause problems in the future, and the behaviour should be the default. However, since there is a wildcard it is worth testing it (as Lars suggested). Otherwise, simply putting the folders:

```bash
isort invenio_app tests --recursive --check-only --diff 
```

Has worked in other repos. In addition, I am not sure about the `.` (meaining you are doing recursive on local + two folders that are already in local. Not sure is needed.

Reply:
Second that: just using the directory name is good enough.

Reply:
Thank you for the comments :) Already made a new commit with the changes! Very useful comments!